packages/sommelier.rb

```diff
@@ -3,7 +3,7 @@
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
-  version '1382ce084cc40'
+  version '1778a8bb16c3'
```

Comment:
Not sure where you got that hash but I was thinking about `version '1382ce084cc407'`.